stable/guardian/values.yaml

```diff
@@ -59,6 +59,22 @@ app:
           - job
           - run
           - appeal_expiration_revocation
```

Comment:
We need to update the appeal expiration related jobs also to grant related ones.

Reply:
updated